rust-code-analysis-web/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "rust-code-analysis-web"
+version = "0.0.18"
+authors = ["Calixte Denizet <cdenizet@mozilla.com>"]
+edition = "2018"
+keywords = ["metrics"]
```

Comment:
We should improve these keywords. I've filed a follow-up: #375.

Reply:
Thank you! :)